src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -38,11 +39,15 @@ class EntityReferenceItemNormalizer extends FieldItemNormalizer implements UuidR
    *
    * @param \Drupal\hal\LinkManager\LinkManagerInterface $link_manager
    *   The hypermedia link manager.
-   * @param \Drupal\hal\EntityResolver\EntityResolverInterface $entity_Resolver
+   * @param \Drupal\serialization\EntityResolver\EntityResolverInterface $entity_Resolver
    *   The entity resolver.
+   * @param \Drupal\jsonld\ContextGenerator\JsonldContextGeneratorInterface $jsonld_context
+   *   The Json-Ld context service.
    */
-  public function __construct(LinkManagerInterface $link_manager, EntityResolverInterface $entity_Resolver) {
-
+  public function __construct(LinkManagerInterface $link_manager,
+  EntityResolverInterface $entity_Resolver,
+                              JsonldContextGeneratorInterface $jsonld_context) {
+    parent::__construct($jsonld_context);
```

Comment:
Because the `$this->jsonldContextGenerator` is initialized in the `FieldItemNormalizer` which this class extends. So that variable is already handled in the parent class constructor.